commonknowledge/mapped
# fix: squash some console errors when adding a data source and viewing a report

## Description
Simply squashes a few console errors.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-501/clear-console-errors

## How Can It Be Tested?
- Add a new data source
- Create a new report with that data source
- Observe no errors in the JS console.
